part_1/title_I/sekcja_2_PL.md

```diff
@@ -0,0 +1,27 @@
+### <a name="section_2"></a><p align="center">Artyku� 2</p>
+
+Definicje niekt�rych poj��
+
+(1) Rodzaje broni i amunicji s� okre�lone w za��czniku nr 1 do tej ustawy.
+
+(2) Na potrzeby niniejszej Ustawy rozumie si� nast�puj�co:
+
+a) przez posiadanie broni i amunicji rozumie si�
+
+1. bro� lub amunicj� w obr�bie lokalu mieszkalnego, lub handlowego _[miejsca prowadzenia dzia�alno�ci, przyp. t�umacza]_, lub wewn�trz wyznaczonego, wyra�nie odgraniczonego obszaru nieruchomo�ci za zgod� w�a�ciciela lub dzier�awcy tych lokali lub nieruchomo�ci;
+
+2. bro� roz�adowana bez naboj�w w magazynku nabojowym wymiennym, magazynku nabojowym sta�ym [wbudowanym, przyp. t�umacza], komorze nabojowej [cz�ci lufy z komor� nabojow� pistoletu, karabinu lub strzelby, przyp. t�umacza], lub kom�r nabojowych w cylindrze rewolweru i umieszczonych w zamkni�tym pojemniku przeznaczonym do transportu broni z miejsca na miejsce.
+
+b) przez noszenie broni i amunicji rozumie si� mie� bro� lub amunicj� przy sobie z wyj�tkiem przypadk�w, o kt�rych mowa w lit. (a).
+
+c) przez miejsce zamieszkania rozumie si� adres miejsca sta�ego pobytu obywatela Republiki Czeskiej<a name="fn4_ref"></a><sup>[4)](#fn4)</sup> lub adres sta�ego lub czasowego pobytu<a name="fn5_ref"></a><sup>[5)](#fn5)</sup> cudzoziemca.<a name="fn6_ref"></a><sup>[6)](#fn6)</sup>
+
+d) przez przedsi�biorc� z sektora zbrojeniowego rozumie si� dowoln� osob� fizyczn� lub osob� prawn�, kt�ra w ramach rozporz�dzenia:<a name="fn7_ref"></a><sup>[7)](#fn7)</sup>
+
+1. rozwija, produkuje, przerabia, naprawia, deaktywuje [pozbawia cech u�ytkowych, przyp. t�umacza] lub niszczy bro� lub amunicj�;
+
+2. przechowuje, magazynuje, wypo�ycza, transportuje, kupuje, sprzedaje lub przyjmuje zam�wienia kupna i sprzeda�y ; lub
+
+3. po�redniczy w nabywaniu lub zbywaniu broni lub amunicji [przez inne osoby lub podmioty, przyp. t�umacza].
+
+e) przez porzucenie broni i amunicji rozumie si� umo�liwienie innej osobie faktycznego _[fizycznego, przyp. t�umacza]_ rozporz�dzania _[dysponowania, przyp. t�umacza]_ broni� i amunicj�. _[w tek�cie angielskim w miejscu "porzucenie" wyst�puje "providing", co t�umaczy�oby si� na "udost�pnianie" lub "dostarczanie", przyp. t�umacza]_
```

Comment:
Jest takie prawne sformułowanie "faktická námitka → práv. objection in point of fact", [ze słownika do angielskiego](https://slovnik.seznam.cz/cz-en/?q=fakticky) można wywnioskować o faktycznym (rzeczywistym, de facto....)